tests/behat/features/fields.feature

```diff
@@ -112,6 +107,15 @@ Feature: Fields for different Media types
     And I should not see an "label[for=edit-field-media-topic-0-target-id].required" element
     And I should see an "input#edit-field-media-topic-0-target-id" element
 
+  @api @javascript @nosuggest @skipped
```

Comment:
@vincent-gao Could you please cherry-pick change for https://digital-engagement.atlassian.net/browse/SDPA-3802. The scenario name could be changed so it relates to the test rather than ClamAV i.e. Test file uploads

Reply:
thanks @GROwen